admin/orders.php

```diff
@@ -115,6 +115,14 @@
         $oID = zen_db_prepare_input($_GET['oID']);
         $comments = zen_db_prepare_input($_POST['comments']);
         $status = (int)zen_db_prepare_input($_POST['status']);
+        $order = new order((int)$oID);
+        $emailLanguageQuery = $db->Execute("SELECT directory
+                                        FROM " . TABLE_LANGUAGES . "
+                                        WHERE languages_id = " . (int)$order->info['language_id']);
+        $emailLanguage = $emailLanguageQuery->fields['directory'];
+
+        require(DIR_WS_LANGUAGES . $emailLanguage . '/orders_email.php');
+
```

Comment:
I was looking for something like this, but did not know the function `get_language_data_by_id()`, that is much cleaner.
I will make this change, as also the change to `zen_load_language_file()` in the `functions_helpers.php`